docs/source/conf.py

```diff
@@ -72,7 +72,7 @@
 
 # General information about the project.
 project = 'AiiDA'
-copyright = '2019, ECOLE POLYTECHNIQUE FEDERALE DE LAUSANNE (Theory and Simulation of Materials (THEOS) and National Centre for Computational Design and Discovery of Novel Materials (NCCR MARVEL)), Switzerland and ROBERT BOSCH LLC, USA. All rights reserved'
+copyright = '2020, ECOLE POLYTECHNIQUE FEDERALE DE LAUSANNE (Theory and Simulation of Materials (THEOS) and National Centre for Computational Design and Discovery of Novel Materials (NCCR MARVEL)), Switzerland and ROBERT BOSCH LLC, USA. All rights reserved'
```

Comment:
can you also update `LICENSE.txt` please?

Reply:
I believe that it is generally not necessary to update the year in these kind of notices, but do we have some kind of guideline for this? Should it say `2014-2020` instead?

Reply:
LICENSE.txt updated. I agree that it's better to put the starting date as well, so I've now added it (set to 2014 both here and in LICENSE.txt)